src/lennybot/check/docker_image_available.py

```diff
@@ -180,7 +180,12 @@ def _exists_on_registry(self, image: DockerImage, access_token: Optional[str] =
 
         request_url = f"https://{image._registry}/v2/{image._name}/manifests/{image._tag}"
 
-        headers = {"Accept": "application/vnd.oci.image.index.v1+json"}
+        # depending on the registry it my helps adding the write accept header :)
+        # https://github.com/goharbor/harbor/issues/16075
+        headers = {
+            "Accept": "application/vnd.oci.image.index.v1+json",
+            "Accept": "application/vnd.docker.distribution.manifest.list.v2+json",
+        }
```

Comment:
Whats the result of
```
python3 -c 'headers = {"Accept": "application/vnd.oci.image.index.v1+json", "Accept": "application/vnd.docker.distribution.manifest.list.v2+json"};print(headers)'
```

Reply:
See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept for more information